packages/node/vrdt-common/src/rest/VroRestClient.ts

```diff
@@ -321,8 +321,7 @@ export class VroRestClient {
             "GET",
             `workflows/${workflowId}/executions/${executionId}/syslogs` +
                 `?conditions=severity=${severity}` +
-                `&conditions=timestamp${encodeURIComponent(">")}${timestamp}` +
-                "&conditions=type=system"
+                `&conditions=timestamp${encodeURIComponent(">")}${timestamp}` // + "&conditions=type=system"
```

Comment:
I think we can remove the commented condition, probably we won't need it anymore